test/events/impression.spec.ts

```diff
@@ -180,6 +180,7 @@ describe('impression tests', () => {
                     //title: 'boats',
                     sku: 'asdfasf98',
                     price: 100.21,
+                    currency: 'cad',
```

Comment:
Since "currency" is being added to all product-oriented event types, we should make sure we update the tests for every event type.

Also, don't forget about the integration tests too (the tests written in JavaScript that use Puppeteer).